spec/system/admin/orders/having_producer_products/hub_actions_spec.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+RSpec.describe '
+  As a hub (producer seller) who have the ability to update
+  orders having their products
+' do
+  include AdminHelper
+  include AuthenticationHelper
+  include WebHelper
+
+  let!(:hub1) { create(:distributor_enterprise, name: 'My hub1') }
+  let!(:hub1_v1) { create(:variant, supplier_id: hub1.id) }
+  let!(:hub1_v2) { create(:variant, supplier_id: hub1.id) }
```

Comment:
These may be simplified:
```suggestion
  let!(:hub1_v1) { create(:variant, supplier: hub1) }
  let!(:hub1_v2) { create(:variant, supplier: hub1) }
```

Also with `variant_id` below